test/fixtures/mson/enum-multiple-default.json

```diff
@@ -75,6 +88,19 @@
                       "value": {
                         "element": "enum",
                         "attributes": {
+                          "enumerations": {
+                            "element": "array",
+                            "content": [
+                              {
+                                "element": "string",
+                                "content": "J"
+                              },
+                              {
+                                "element": "string",
+                                "content": "K"
+                              }
+                            ]
```

Comment:
This matches any `string`. Are you missing `fixed` attribute on the `string` elements above?